Makefile

```diff
@@ -5,5 +5,17 @@ MIN_JS:=static/js/app.min.js
 JS_SRC:=$(shell find static/js \! -name '*.min.*' -name '*.js')
 MINIFY:=$(shell which uglifyjs >/dev/null && echo `which uglifyjs` || echo cat)
 
+# Declarations for SCSS
+SCSS_SRC:=$(shell find static/scss -name '*.scss')
+PYSCSS:=$(shell which pyscss >/dev/null && echo `which pyscss` || echo cat)
+
 $(MIN_JS): $(JS_SRC)
 	$(MINIFY) $^ > $@
+
+scss:
+	for i in $(shell ls static/scss/); do \
+		echo "Making $$i";\
+		$(PYSCSS) -o static/css/$${i%.scss}.css static/scss/$$i;\
```

Comment:
cat -o = invalid option -- o

since it's a deploy script, it's fine to error out if it's not installed IMO